### DISCUSSION TITLE
Argocd with Chart Musueum

### DISCUSSION THREAD

**Discussion by KeerthigaKrishnan** [argoproj/argo-cd#5487] [Q&A]
Hi Team, 

Do you have any timeline or any future plan for supporting ArgoCd from Chartmusuem?. If it is already there, please provide the docs. I am unable to find online.

**Comment by jannfis** [comment]
Can you please elaborate a little more on your question, @KeerthigaKrishnan?

If you are asking whether it's possible to install Helm charts from a Helm repository (such as Chart Museum), then it is already possible as described here: https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#helm-chart-repositories

It is not well documented, but you can also connect such a repository in the CLI by using `--type helm` to `argocd repo add` or by using the web UI and setting the repository type to "Helm" instead of "Git", as shown here:

![image](https://user-images.githubusercontent.com/3942683/107610856-e3c65780-6c42-11eb-977b-2558b70a17c0.png)

**Comment by KeerthigaKrishnan** [comment]
Hi,

Thanks for the info. This is great..

Regards
K.Keerthiga
